test/excludes/FixturesTest.rb

```diff
@@ -1,2 +1,11 @@
 exclude :test_bulk_insert_multiple_table_with_a_multi_statement_query, "Skipping until we can triage further. See https://github.com/cockroachdb/activerecord-cockroachdb-adapter/issues/48"
 exclude :test_bulk_insert_with_a_multi_statement_query_in_a_nested_transaction, "Skipping until we can triage further. See https://github.com/cockroachdb/activerecord-cockroachdb-adapter/issues/48"
+exclude :test_clean_fixtures, "Skipping"
+exclude :test_auto_value_on_primary_key, "Skipping"
```

Comment:
can you make the comment say "Skipping the PostgreSQL test, but reimplemented for CockroachDB in test/cases/fixtures_test.rb"

Reply:
Done